components/Snackbar/src/MDCSnackbarManager.h

```diff
@@ -51,6 +51,11 @@
  */
 @interface MDCSnackbarManager : NSObject
 
+/**
+ An instance of MDCSnackbarManager bound to the main screen.
```

Comment:
Is the "bound to the main screen" statement relevant to a client? Can they change which screen the manager is bound to?

Reply:
I looked a bit deeper, and it doesn't even default to the main screen of an application. It searches for any MDCOverlayWindow (hidden or not) and uses the first one it finds. If it can't, it tries the app delegate's window, and if that doesn't exist, picks the key window.